tests/dom/basictests.cpp

```diff
@@ -1351,6 +1381,8 @@ namespace dom_api_tests {
            array_iterator_empty() &&
            object_iterator_advance() &&
            array_iterator_advance() &&
+		   convert_object_to_element() &&
```

Comment:
Here the new code is not indented like the previous and following lines. We do not enforce style or strict rules, but it should be trivial to fix...

Reply:
I'm sorry I didn't catch this issue sooner. I've fixed it now.

Reply:
That's fine. We do not have rules to that effect so your code was fine.